src/main.rs

```diff
@@ -70,6 +69,19 @@ Options:
   --skip-readme             Do not attempt to process README.md files listed in Cargo.toml manifests.
 "#;
 
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, Deserialize)]
+enum CheckerType {
+    #[serde(alias = "hunspell")]
```

Comment:
```suggestion
    #[serde(alias = "hunspell")]
    #[serde(alias = "Hunspell")]
```

Reply:
It's already named `Hunspell`, so that is already covered iiu the serde docs correctly.